modules/actions/types/worker.go

```diff
@@ -94,17 +94,9 @@ func (w *ActionsWorker) handleError(writer http.ResponseWriter, path string, err
 }
 
 // Start starts the worker
-func (w *ActionsWorker) Start(port uint) {
```

Comment:
@ankurdotb linter might be failing because of this. One of the rules require us to set timeouts on connections